client/plasmacli/eth/prove.go

```diff
@@ -0,0 +1,98 @@
+package eth
```

Comment:
Why is this in `eth`?

Reply:
I wanted to keep proof retrieval in prove.go, if prove.go was in plasmacli/ I wouldn't be able to import it since it would be in a program package. Given that prove.go is only useful for eth cmds, I figured it would be best to keep it there for now even though it lies under the plasmacli root cmd. I could isolate the getProof function and add it to file in client/